src/rgw/store/rados/rgw_bucket.cc

```diff
@@ -2066,7 +2066,7 @@ class RGWMetadataHandlerPut_BucketInstance : public RGWMetadataHandlerPut_SObj
                                        RGWSI_MetaBackend_Handler::Op *_op, string& entry,
                                        RGWMetadataObject *_obj, RGWObjVersionTracker& objv_tracker,
 				       optional_yield y,
-                                       RGWMDLogSyncType type, bool from_remote_zone) : RGWMetadataHandlerPut_SObj(_handler, _op, entry, obj, objv_tracker, y, type, from_remote_zone),
+                                       RGWMDLogSyncType type, bool from_remote_zone) : RGWMetadataHandlerPut_SObj(_handler, _op, entry, nullptr, objv_tracker, y, type, from_remote_zone),
```

Comment:
why do you pass null?
isn't the correct value should be the one coming from `_obj`?

Reply:
has been changed to `_obj`. PTAL

> why do you pass null? isn't the correct value should be the one coming from `_obj`?